src/compensations/Compensations.tsx

```diff
@@ -52,9 +57,27 @@ const Compensations = ({ compensations }: CompensationsProps) => {
     );
   };
 
+  const updateLocation = (id: string) => {
+    setSelectedLocation(id);
+  };
+
+  const locationOptions: IOption[] = Object.values(locations).map(
+    (companyLocation) => ({
+      id: companyLocation.companyLocationName.toLowerCase(),
+      label: companyLocation.companyLocationName,
+    }),
+  );
+
   return (
     <div className={styles.wrapper}>
       <Text type="h1">{compensations.basicTitle}</Text>
+      <RadioButtonGroup
```

Comment:
Does it make sense to display the location radio group if the salary calculator is not visible?

Reply:
Yes, because we still want to display benefits!